src/database/server.js

```diff
@@ -0,0 +1,23 @@
+const jsonServer = require('json-server');
+
+const server = jsonServer.create();
+const router = jsonServer.router('./src/database/db.json');
+const middlewares = jsonServer.defaults();
+
+function isAuthorized(req) {
+  const { login, password } = req.body;
+  if (login === 'rob' && password === 'rob') return true;
```

Comment:
Authorization should compare user login and password stored in the database.
Of course in the real application, passwords should not be kept in plain text.
https://highedwebtech.com/2017/04/03/password-security-plaintext/
https://plaintextoffenders.com/

Bonus points if you implement hashing passwords and then storing them in the DB.
https://github.com/kelektiv/node.bcrypt.js